koocook_core/support/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+
+class CustomisableForm(forms.ModelForm):
+    customised_field = tuple()
+
+    def __init__(self, *args, **kwargs):
+        if 'user' in kwargs:
+            self.user = kwargs.pop('user', None)
+        super().__init__(*args, **kwargs)
+        # self.fields['author'].disabled = True
+
+    @property
+    def vanilla_fields(self):
+        return [field for field in self if not (field.name in self.customised_field or field.is_hidden)]
```

Comment:
I would suggest using American English to prevent the same mistake as `SerialisableModel` but if it's a lot of trouble this is fine I guess.

Reply:
Are there any conventions of this? I've found some popular sources suggesting AE, but our project won't be passed or built on by others anyway. So I didn't see the need of AE in the codebase.

Reply:
Normal standards use American English. This is also seen in HTML and CSS. But smaller non-international projects sometimes use British English. The general consensus is still American English though. This is why IDEs flag non-american variants.